server/itx/subchain_test.go

```diff
@@ -7,10 +7,20 @@
 package itx
```

Comment:
You could ignore subchain related, which might be deprecated.

Reply:
sure